src/sql/store.py

```diff
@@ -104,7 +104,7 @@ def __str__(self) -> str:
         replace to the ' (backtick) symbol if it's supported
         """
         with_clause_template = Template(
-            """WITH{% for name in with_ %} "{{name}}" AS ({{saved[name]._query}})\
+            """WITH{% for name in with_ %} {{name}} AS ({{saved[name]._query}})\
```

Comment:
@yafimvo 
Something needs your review

Do you remember in the previous [Drop sqlglot for generating CTEs](https://github.com/ploomber/jupysql/pull/318) you asked [Is there any chance the default identifier (") won't work?](https://github.com/ploomber/jupysql/pull/318#pullrequestreview-1364536490)?

We met the problem you had in this snowflake integration testing after rebasing with that PR: The Snowflake doesn't support `WITH "CTE_NAME" AS` (double quote on CTE_NAME). However, I am able to fix this issue after removing the double quote in WITH

So we get the conclusion: We can't say the dialect support " (double quote) if it doesn't support ` (backtick)

I think if removing double quote is the most generous case then we can just not use the double quote in basic template

LMK what do you think

Reply:
If you agree I can go ahead to update comments & related code sections